typescript/token/src/config.ts

```diff
@@ -20,6 +20,8 @@ export type ERC20Metadata = TokenMetadata & {
   decimals: number;
 };
 
+export type MinimalTokenMetadata = Omit<ERC20Metadata, 'totalSupply'>;
```

Comment:
I think both erc20 and 721 have total supply

Reply:
Yeah but there are a bunch of places where we need an interface comprised of `name, symbol, decimal`, especially across the UI. 

Should I rename? Not sure what else to call it